reverse-proxy/nginx.conf

```diff
@@ -0,0 +1,22 @@
+worker_processes 1;
```

Comment:
For this I think its suggested to run it as "auto"
Hard coding 1 sound wrong but I can't find docs to what a proper node configuration would be

Reply:
see where for 
`worker_processes auto;`
https://gist.github.com/denji/8359866